gnd/src/main/java/com/google/android/gnd/rx/RxCompletable.java

```diff
@@ -45,4 +49,17 @@ public static Completable completeOrError(
           }
         });
   }
+
+  public static Single<Boolean> toSingle(
```

Comment:
This method is actually doing a lot more than just converting the Completable to a Single; it's returning whether the Single completed successfully, and is causing a side effect on the provided Consumer. I'd suggest a few improvements to make this clearer:

* Give the method a more explicit name, like `toBooleanSingle()` to indicate this is doing something special.
* Call the callback param for when it's actually invoked (`onError`).
* If this is every called with null `exceptionConsumer` (I didn't see it on a quick scan), create a separate override of this method without the Consumer (i.e. toBooleanSingle(Completable) and toBooleanSingle(Completable, onErro).
*  Add minimal Javadoc explaining these semantics (esp. since the method is public().